docs/getting_started/experimental/source_files/consume_files.py

```diff
@@ -0,0 +1,10 @@
+import os

```

Comment:
is it used in any notebook?

Reply:
I missed one file to put.  I'm adding manage_data_directory.ipynb.